v2/sourcedb-to-spanner/README_Sourcedb_to_Spanner_Flex.md

```diff
@@ -278,21 +274,20 @@ resource "google_dataflow_flex_template_job" "sourcedb_to_spanner_flex" {
     projectId = "<projectId>"
     outputDirectory = "<outputDirectory>"
     # sourceDbDialect = "MYSQL"
-    # jdbcDriverJars = "gs://your-bucket/driver_jar1.jar,gs://your-bucket/driver_jar2.jar"
+    # jdbcDriverJars = ""
```

Comment:
Could we add "<gsPathToDriverJars>" here?

Reply:
Re-generating the readme docs fixed the problem - https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/2062